pkg/operator/ceph/cluster/watcher.go

```diff
@@ -56,6 +56,7 @@ var (
 	nodesCheckedForReconcile = sets.New[string]()
 	networkFenceLabel        = "cephClusterUID"
 	errActiveClientNotFound  = stderrors.New("active client not found")
+	enableCephFsFencing      = false
```

Comment:
what is the use of this flag as this cannot be set to `true` without code changes? may be a comment on the code for now?

The above is just a suggestion, LGTM to disable cephfs fencing.

Reply:
added the comment